.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml

```diff
@@ -115,7 +115,7 @@ steps:
           queue: n2-4-spot
         depends_on: build
         timeout_in_minutes: 60
-        parallelism: 4
```

Comment:
I've checked the serverless rule management prebuilt rules execution and now is executing tests in the 4 parallels, so I'll keep it as it is instead of moving to 2. Teams are working on adding more tests and fixing broken ones so I'll keep the 4 parallels.

Reply:
Fixed with af3684fdd3a5f18f7ddb1bd729f4f3760af7b13d